Floofbot/Modules/Helpers/EightBall.cs

```diff
@@ -0,0 +1,36 @@
+using System;
+using System.Collections.Generic;
+
+namespace Floofbot.Modules.Helpers
+{
+    public static class EightBall
+    {
+        public static string Response() {
+            var responses = new List<string> {
```

Comment:
We can make responses a `private static readonly List<string>` variable within this class now!

Reply:
maybe rename this to GetRandomResponse()